src/nyc_trees/apps/survey/templates/survey/reserve_blockface.html

```diff
@@ -11,13 +11,7 @@
       <h2>Reservations</h2>
       {% include 'survey/partials/reservations_legend.html' %}
       <div class="action-bar reservations-action-bar">
-          {% if reservations.current > 0 %}
-          <div id="reservations-start">
-              <a class="btn btn-primary pull-left" href="{% url 'survey' %}">Start Mapping</a>
-              <a id="reservations-add-remove" class="btn btn-primary pull-right" href="javascript:">Add / Remove</a>
-          </div>
-          {% endif %}
-          <div id="reservations-cart-action" class="reservation-action-item {% if reservations.current == 0 %}active{% else %}hidden{% endif %}">
+          <div id="reservations-cart-action" class="active">
```

Comment:
This needs to retain the class `reservation-action-item` (in addition to `active` or there will be a styling regression)